Source/Fuse.Marshal/Marshal.Convert.uno

```diff
@@ -76,13 +76,31 @@ namespace Fuse
 			return false;
 		}
 
+		/**
+			Be aware this function may throw a NullReferenceException if the type cannot be converted to the desired one. It is advised to use TryToType or TryConvertTo instead.
```

Comment:
It was throwing NullReferenceException since we convert to null and then attempt to cast, which results in NRE. I did't wish to change the odd behaviour in this PR.